src/backend/cuda/nvrtc/launch_args.hpp

```diff
@@ -0,0 +1,55 @@
+/*******************************************************
+ * Copyright (c) 2019, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#pragma once
+
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include <nvrtc.h>
```

Comment:
I don't think this header is needed here.

Reply:
We have `CUStream` and `CUEvent` usage in that header.